jakartaee10/JakartaEE10ReleasePlan.md

```diff
@@ -4,3 +4,167 @@
+ * spec module name is `jakarta.<specName>`
+ * spec provides JPMS module definition (reliance on automatic modules is not allowed)
+ * spec updates provider lookup algorithm to include search on module-path (if applicable)
+ * TCK tests should be updated or provided to ensure a properly configured and functional `module-info.class`
+* The Platform and Web Profile Specification API will not provide a platform/profile level `module-info.class` for EE 10
+ * Don’t want to set the expectation that implementations must support JPMS at this point
+ * Implementations can still experiment with their own aggregator modules
+
+## “Removal” of Two EJB Features
+The number of compatible implementations available for specification ratification is limited due to the requirement that all optional features need to be implemented and verified via the TCK.
+This is a current restriction due to the EFSP and our Platform specification.
+The EFSP is being worked on via [this issue](https://github.com/EclipseFdn/EFSP/issues/22) and should be resolved in the next iteration of the EFSP.
+
+For the Platform specification, the following two features will be tagged as “removed” or “deprecated” (exact term and/or mechanism still needs to be determined).
+* EJB Entity Beans (CMP and BMP)
+* Embeddable EJB Container
+
+The idea behind this tagging is to indicate that these features will no longer be required for specification ratification.
+These features will still be documented as “optional” in the EJB specification.
+And, the TCKs will still be available for verifying implementations that choose to implement these features.
+The Platform specification will be updated to properly reflect that these features will not be required for ratifying compatible implementations.
+
+## Standalone Component TCKs ([issue](https://github.com/eclipse-ee4j/jakartaee-platform/issues/333))
+Each Specification project should own the TCK source and produce the TCK artifacts.
+This needs to be done in such a way that the artifacts can be run to validate implementations at the individual specification level, as well as integrated into platform level TCKs.
+
+This is a run-at goal for Jakarta EE 10.
+We know that not all Specifications can establish their standalone TCKs in this timeframe.
+The Platform TCK can initiate this refactoring by breaking apart the TCK source into separate directories within the Platform TCK repository.
+This will help with the eventual move of these TCK directories into their respective component repositories.
+
+## Component Specification Project Releases
+As mentioned, many of the Component Specification projects have already declared their intent for either a Major or Minor version update for Jakarta EE 10.
+These projects have already submitted or completed their Plan Reviews for inclusion in Jakarta EE 10.
+These updates are outlined in the sections below.
+
+## Deliverables in Jakarta EE 10
```

Comment:
I would add a statement saying that the expected timeframe for individual specification final versions is Oct of 2021 as previously communicated.

Reply:
I will add a blurb about 4Q2021 since that's what we have talked about and not specifically Oct 2021.